packages/video_player/tizen/src/message.h

```diff
@@ -1,5 +1,5 @@
-#ifndef VIDEO_PLAYER_MESSAGE_H_
-#define VIDEO_PLAYER_MESSAGE_H_
+#ifndef FLUTTER_PLUGIN_VIDEO_PLAYER_MESSAGE_H_
```

Comment:
This is a question, is there a reason to prefix header guard with `FLUTTER_PLUGIN`? Should other plugins also do this?

Reply:
There's not much reason as it's not a public header and thus there's little chance of name conflicts. It's just a convention of this repo. It's not a strict rule so it's up to you.